apps/backoffice-v2/src/pages/Entity/components/Notes/Note.tsx

```diff
@@ -0,0 +1,55 @@
+import dayjs from 'dayjs';
+import * as React from 'react';
+import { useMemo } from 'react';
+import DOMPurify from 'dompurify';
+
+import { TNote } from './types';
+import { TUsers } from '@/domains/users/types';
+import { Tooltip } from '@/common/components/atoms/Tooltip/Tooltip';
+import { UserAvatar } from '@/common/components/atoms/UserAvatar/UserAvatar';
+import { TooltipTrigger } from '@/common/components/atoms/Tooltip/Tooltip.Trigger';
+import { TooltipContent } from '@/common/components/atoms/Tooltip/Tooltip.Content';
+
+export const Note = ({ content, createdAt, user }: TNote & { user: TUsers[number] }) => {
+  const prettyDate = useMemo(() => {
+    const localDateTime = dayjs.utc(createdAt).local();
+
+    const date = localDateTime.format('MMM DD, YYYY');
```

Comment:
Pretty sure this will work.
```suggestion
    const date = localDateTime.format('MMM DD, YYYY, HH:mm');
```